src/overlays/actors/ovl_Bg_Hidan_Sekizou/z_bg_hidan_sekizou.c

```diff
@@ -312,14 +312,15 @@ Gfx* func_8088D9F4(PlayState* play, BgHidanSekizou* this, s16 arg2, MtxF* arg3,
     gDPSetPrimColor(arg7++, 0, 1, 255, 255, 0, 150);
     gDPSetEnvColor(arg7++, 255, 0, 0, 255);
 
-    arg3->xx = arg3->yy = arg3->zz = (0.7f * phi_f12) + 0.5f;
+    mtx2 = mtx;
+    mtx2->xx = mtx2->yy = mtx2->zz = (0.7f * phi_f12) + 0.5f;
 
-    temp_f2 = (arg3->xx * 10.0f * phi_f12) + 20.0f;
-    arg3->xw = (temp_f2 * arg4) + this->dyna.actor.world.pos.x;
-    arg3->yw = this->dyna.actor.world.pos.y + 30.0f + (.7f * phi_f12);
-    arg3->zw = (temp_f2 * arg5) + this->dyna.actor.world.pos.z;
+    temp_f2 = ((mtx2->xx * 10.0f) * phi_f12) + 20.0f;
+    mtx2->xw = (temp_f2 * arg4) + this->dyna.actor.world.pos.x;
+    mtx2->yw = (this->dyna.actor.world.pos.y + 30.0f) + (.7f * phi_f12);
```

Comment:
these added parens are needed?

Reply:
ah yeah this is a vestige of other head-bashing